pkg/model/piped_stat.proto

```diff
@@ -0,0 +1,25 @@
+// Copyright 2021 The PipeCD Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package pipe.model;
+option go_package = "github.com/pipe-cd/pipe/pkg/model";
+
+import "validate/validate.proto";
+
+message PipedStat {
```

Comment:
How about adding PipedID into this message?
I know we are having that ID from the map key but ID inside its data could be helpful as well.

Reply:
Sure 👍 But I'm thinking about the current model name, just want to make it PipedMetrics. In that case, PipedMetrics.PipedID and PipedMetrics.Stat looks better than PipedStat.PipedID and PipedStat.Stat as currently. Just wonder is that name too generics or something 🤔 How do you think about that?

Reply:
I think `PipedMetrics` is good too. But currently, our RPC name is `Report...Stats`, right?
So maybe we should keep that model name as `PipedStat` and change its `Stat` field to `Metrics`.
Or we can reuse the old `PipedStats` model, deprecate unused fields and add new fields like `PipedID`, `Metrics`.

Reply:
> So maybe we should keep that model name as PipedStat and change its Stat field to Metrics.

Get your point, lets me adopt this 👍 ( Personally don't think the name should be `XXXStats` since this message covers only a single value of piped committed stat at a time )

Reply:
Addressed by https://github.com/pipe-cd/pipe/pull/2261/commits/ae07febc5c251a1f385208128da01fb9abe14aeb 🙏